app/code/core/Mage/Core/functions.php

```diff
@@ -178,6 +160,11 @@ function mageCoreErrorHandler($errno, $errstr, $errfile, $errline){
         define('E_DEPRECATED', 8192);
     }
 
+    // Suppress deprecation warnings on PHP 7.x
+    if ($errno == E_DEPRECATED && version_compare(PHP_VERSION, '7.0.0', '>=')) {
```

Comment:
we should think about removing this again at a later point, to not make future updates harder